infra/modular-examples/gcp-google-workspace/variables.tf

```diff
@@ -143,8 +143,20 @@ variable "google_workspace_example_admin" {
   default     = null # will failover to user
 }
 
-variable "salesforce_domain" {
+variable "msft_tenant_id" {
   type        = string
+  description = "ID of Microsoft tenant to connect to (req'd only if config includes MSFT connectors)"
   default     = ""
+}
+
+variable "msft_owners_email" {
```

Comment:
ATM MSFT connectors are not supported for GCP examples (main needs to create the AAD apps, add the secrets, federation, etc). That was done [here](https://github.com/Worklytics/psoxy/pull/242#discussion_r1089087399) but dropped on the PR as not used yet; if we need it please create a task and I'll do it during this sprint. 
Otherwise I'd not include `msft_tenant_id` as part of variables for GCP